### PR TITLE
bump: boilerplate

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Bumping boilerplate for better logging, etc.

### CODE REVIEW

File: pkg/boilerplate/operator/sync.go [hunk 1]
```diff
@@ -1,9 +1,8 @@
 package operator
 
 import (
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
 	"github.com/openshift/console-operator/pkg/boilerplate/controller"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (enj) [inline pkg/boilerplate/operator/sync.go]: Guess I need to fix this.

**Comment by benjaminapetersen** [comment]
terraform fail....

**Comment by benjaminapetersen** [comment]
(maybe?  😓 )

**Pull request merged**